.github/workflows/ci-pipeline.yml

```
@@ -0,0 +1,16 @@
+name: Continuous Integration
+on: 
+  push:
+    paths-ignore:
+      - 'documentation/**'
+      - 'README.md'
+  pull_request:
+
+jobs:
+  build:
+    name: Build and test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - run: docker build --target test --tag my-test-image .
+    - run: docker run --env-file .env.test my-test-image
```

You could put names on these steps just to make it appear a bit more nicely in the UI.
```suggestion
    - name: Build testing image
      run: docker build --target test --tag my-test-image .
    - name: Run tests
      run: docker run --env-file .env.test my-test-image
```